modules/ROOT/pages/single-sign-on-with-saml.adoc

```diff
@@ -97,7 +97,7 @@ edit the authenticationManager-config.properties as follows:
 Make sure to link:multi-tenancy-and-tenant-configuration[set the right tenant admin username].
  It is recommended to also replace the value of the passphrase (property auth.passphrase) which is used by the engine to verify the authentication request.
  The value must be the same as in the file *bonita-tenant-sp-custom.properties*. +
- If you need users to be able to log in without having an account on the IDP, you can authorize it by setting the property `saml.auth.standard.allowed` to true. Users will then be able to log in using the portal login page (/login.jsp) provided they have a bonita account and their password is different from their username. +
+ If you need users to be able to log in without having an account on the IDP, you can authorize it by setting the property `saml.auth.standard.allowed` to true. Users will then be able to log in using the Bonita login page (/login.jsp) provided they have a bonita account and their password is different from their username. +
  If only a limited group of users need to bypass SAML authentication method you can restrain it by setting the property `saml.auth.standard.allowed` to false and setting the property `auth.tenant.standard.whitelist` with the list of authorized usernames (comma separated).
```

Comment:
SAML authentication method you=> SAML authentication method, you can

Reply:
Added